src/examples/dropdowns/MenuButton.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Dropdown, Menu, Item, Trigger } from '@zendeskgarden/react-dropdowns';
+import { Button } from '@zendeskgarden/react-buttons';
+
+const Example = () => {
+  return (
+    /* eslint-disable no-alert */
+    <Dropdown onSelect={item => alert(item)}>
```

Comment:
@rossmoody any ideas/suggestions here?

I can remove the alert. I think alert's main purpose is to demonstrate that a Menu pattern (not component) has list of choices that are actions or functions, which differentiates the Menu usage when used in a Select dropdown.

Reply:
To Henry's point, for this specific example I think the alert is helpful (unless there is a better alternative for illustrating the items prompt an action).

Reply:
I don't want to stand in the way of this one. But I feel like we can do better. Even a `Well` off to the side showing the selection would be less abrupt.

Reply:
Hmmm, the alert after clicking an option in a Menu to show that clicking an option in a menu is performing an action seems appropriate to me for one example trying to show that behavior. If it's too much, I think my vote would be to nix the `onSelect` before introducing another element in the example.

Reply:
I don't think we would want to nix `onSelect` behavior—we want to demonstrate what these things do, not just what we look like. 

The alerts serve a purpose, giving you feedback that what you did was successful, but it's a blunt force approach. Not super inline with how Garden tends to value UX. 

@rossmoody why do you think it's bad to introduce another element into the example?

Reply:
Henry and I were hoping we keep the code example simple to show an example of the component implementation in a basic way, instead of importing the Notification package and having the component code inside this example. I think my priority at this point is to do what the group wants but I don't know that is. Should I try to design some implementations using a Well?

Reply:
And I know this site is a new beginning, but isn't there alert code all over our component [examples in the current Garden site?](https://share.getcloudapp.com/geuw0jRm) Are we sure we can't allow this one component example to get a pass for the sake of simplicity?